src/style.css

```diff
@@ -0,0 +1,148 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+body {
+  margin: 0;
+  padding: 0;
+
+  --moyo-color: #00acdc;
+  --reset: 0;
+  --width-content: 166px;
+  --text-color-1: #060b35;
+  --text-color-2: #616070;
+  --white-color: #fff;
+}
+
+.main {
+  font-family: "Roboto", sans-serif;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 198px;
+  height: 406px;
+  background: var(--white-color);
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__img {
+  width: 160px;
+  height: 134px;
+  margin: 32px 0 40px;
+}
+
+.card__info {
+  margin-bottom: 16px;
+}
+
+.card__title {
+  margin: var(--reset);
+  padding: var(--reset);
+  width: var(--width-content);
+  height: 36px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--text-color-1);
+}
+
+.card__code {
+  margin: 4px 0 0;
+  padding: var(--reset);
+  width: 98px;
+  height: 14px;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--text-color-2);
+}
+
+.card__review {
+  width: var(--width-content);
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  color: var(--text-color-1);
+}
+
+.card__review-rating {
+  margin: 0;
+  padding: 0;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+}
+
+.card__price {
+  margin: 24px 0 16px;
+  width: var(--width-content);
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  color: var(--text-color-1);
+}
+
+.card__price-text {
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--text-color-2);
+}
+
+.card__price-price {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+}
+
+.card__button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: var(--width-content);
+  height: 40px;
+  background: var(--moyo-color);
+  border-radius: 5px;
+  text-decoration: none;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  color: var(--white-color);
+}
+
+.card__button:hover {
+  width: 164px;
+  height: 38px;
+  background: var(--white-color);
+  color: var(--moyo-color);
+  border: 1px solid var(--moyo-color);
+}
+
+/* #region stars */
+.stars {
+  display: flex;
+}
```

Comment:
Move these styles to a separate file
![image](https://user-images.githubusercontent.com/96409563/220564863-c6fe0a8f-9ea6-434c-974f-81e1853e50ea.png)